src/openforms/scss/components/builder/_radio.scss

```diff
@@ -0,0 +1,3 @@
+.vRadioLabel {
+    padding: 0 0 0 5px !important;
```

Comment:
Not sure if this is the right place, but I noticed the padding of labels for radio buttons was off

Reply:
it's not, but I'll take a look why it's off and how the default django styles solve it. possibly there's some markup/classnames missing in the components :)